src/papi_events.csv

```diff
@@ -1864,6 +1864,52 @@ PRESET,PAPI_BR_UCN,NOT_DERIVED,PM_BR_UNCOND
 PRESET,PAPI_BR_TKN,NOT_DERIVED,PM_BR_CORECT_PRED_TAKEN_CMPL
 PRESET,PAPI_FXU_IDL,NOT_DERIVED,PM_FXU_IDLE
 #
+CPU,POWER10
+CPU,power10
+#
+PRESET,PAPI_L1_DCM,DERIVED_ADD,PM_DATA_FROM_L1MISS,PM_ST_MISS_L1
+PRESET,PAPI_L1_LDM,NOT_DERIVED,PM_DATA_FROM_L1MISS
+PRESET,PAPI_L1_STM,NOT_DERIVED,PM_ST_MISS_L1
+PRESET,PAPI_L1_DCW,DERIVED_SUB,PM_ST_FIN,PM_ST_MISS_L1
+PRESET,PAPI_L1_DCR,NOT_DERIVED,PM_LD_HIT_L1
+PRESET,PAPI_L1_DCA,DERIVED_ADD,PM_LD_REF_L1,PM_ST_CMPL
+PRESET,PAPI_L2_DCM,DERIVED_ADD,PM_DATA_FROM_L2MISS,PM_L2_ST_MISS
+PRESET,PAPI_L2_LDM,NOT_DERIVED,PM_L2_LD_MISS
+PRESET,PAPI_L2_STM,NOT_DERIVED,PM_L2_ST_MISS
+PRESET,PAPI_L2_DCR,NOT_DERIVED,PM_DATA_FROM_L2
+PRESET,PAPI_L2_DCW,NOT_DERIVED,PM_L2_ST_HIT
```

Comment:
PRESET,PAPI_L2_DCW,DERIVED_POSTFIX,N0|2|*|,PM_L2_ST_ATTEMPT

Reply:
We haven't modified the Power9 behavior; it was also mapped to PM_L2_ST_HIT in Power9. By default, all stores are written to L2, so for PM_L2_ST_ATTEMPT, our counter will account for both updates and reloads to L2. PM_L2_ST_HIT will count only updates to L2. So, what do you expect from PAPI_L2_DCW?

Reply:
As @jeevitha30 mentioned above, we would like to go with PM_L2_ST_HIT because we expect that reloads should not be accounted.
However, we have multiplied the event result by 2 here also.
PRESET,PAPI_L2_DCW,DERIVED_POSTFIX,N0|2|*|,PM_L2_ST_HIT

Reply:
Yes, the suggested event is good if multiplied by two:
 PRESET,PAPI_L2_DCW,DERIVED_POSTFIX,N0|2|*|,PM_L2_ST_HIT